docs/README.md

```diff
@@ -22,7 +22,7 @@ From this directory:
 	hugo serve
 	```
 
-The site can be viewed at http://localhost:1313
+The site can be viewed at http://localhost:1313/projects/flink/flink-docs-master/
```

Comment:
wouldn't it be nicer to use `hugo -b / serve` to have the docs in root?

Reply:
Mh, that doesn't seem to work

Reply:
`hugo -b "" serve` seems to work

Reply:
Oh great, I couldn't figure it out. I'll add that.

Reply:
`hugo -b localhost:1313` serve would work though